basics/signals/src/index.ts

```diff
@@ -43,11 +45,13 @@ function activate(
   app: JupyterFrontEnd,
   palette: ICommandPalette,
   mainMenu: IMainMenu,
-  launcher: ILauncher | null
+  launcher: ILauncher | null,
+  translator: ITranslator
```

Comment:
Hi @kgoo124 maybe the I translator should go before the "optional" parameter.

`ILauncher | null` I guess this means ILauncher can be `undefined?

Maybe this:
```ts
  translator: ITranslator,
 launcher: ILauncher | null
```

Reads better?

Reply:
Fixed!